pyproject.toml

```diff
@@ -15,7 +15,7 @@ repository = "https://github.com/i-am-bee/beeai-framework-py-starter"
 
 [tool.poetry.dependencies]
 python = ">= 3.11,<4.0"
-beeai_framework = "^0.1.14"
+beeai_framework = {extras = ["duckduckgo", "mcp", "search", "wikipedia"], version = "^0.1.15"}
```

Comment:
`search` is redundant here since it's just an alias for `["duckduckgo", "wikipedia"]`, I'd say either remove `search` or  remove `duckduckgo` and `wikipedia`

Reply:
Updated. We should add `all` group, which would group everything.